libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/UnreadIndicatorAtom.kt

```diff
@@ -29,8 +33,10 @@ fun UnreadIndicatorAtom(
     color: Color = ElementTheme.colors.unreadIndicator,
     isVisible: Boolean = true,
 ) {
+    val contentDescription = stringResource(CommonStrings.a11y_notifications_new_messages)
```

Comment:
`UnreadIndicatorAtom` is also used for [invitation](https://github.com/element-hq/element-x-android/blob/develop/features/home/impl/src/main/kotlin/io/element/android/features/home/impl/components/RoomSummaryRow.kt#L335), so maybe let `contentDescription` be a parameter and add a string `a11y_notifications_new_invitation`?